src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -147,10 +147,6 @@
       <Project>{69e46a6f-9966-45a5-8945-2559fe337827}</Project>
       <Name>RemoteExecutorConsoleApp</Name>
     </ProjectReference>
-    <!-- ToDo: Remove this reference to the live package of System.Globalization once prerelease gets updated again -->
-    <ProjectReference Include="..\pkg\System.Globalization.pkgproj">
```

Comment:
This one I think we should keep, as it is the implementation for this test assembly. If we remove it, then we would be testing a package that we already shipped, instead of the live build.